docs/DeveloperGuide.md

```diff
@@ -548,8 +548,8 @@ unless specified otherwise)
 
 **MSS**
 
-1. User requests to find patients whose names have specified keywords.
-2. UniNurse shows a list of patients whose names have specified keywords.
+1. User requests to find patients whose name, phone number, email, address, tag, task, condition, medication, or remark contains specified keywords.
```

Comment:
Not very sure if we should explicitly state all the details in the use case. Maybe can ask others for opinions?

Reply:
Should keep it general e.g. use "details" instead of listing out everything since future improvements could introduce more details

Reply:
Updated to just "details"